src/PresentationalComponents/RulesTable/helpers.js

```diff
@@ -222,7 +224,7 @@ export const filterConfigItems = (
     },
     {
       label: FC.total_risk.title,
-      type: FC.total_risk.type,
+      type: conditionalFilterType.text,
```

Comment:
We need to make sure these are correct. `FC.total_risk.type` returns "checkbox", this here would return `text`. https://github.com/RedHatInsights/insights-advisor-frontend/blob/master/src/AppConstants.js#L199

Reply:
I'll double check now and fix these. I saw that the type was a string, so I had just replaced that with text.

Reply:
good catch! I've sent up the updated values